merge 2019.06

kkrebs opened the pull request.

kkrebs committed:
MCR-2143 migrate expert search mask to bootstrap 4

kkrebs committed:
MCR-2147 check if name is empty after normalization and remove

duplicate trim() - its already part of normalize()

Possommi committed:
MCR-2148 coordinates_round_second_part fixed bracket position

chiliego committed:
MCR-2074 MCRURNGranularRESTService does not register urn for landing page after urn assignment

chiliego committed:
MCR-2131 When replacing a file with an urn in a derivate the urn is removed from the derivate xml

sebhofmann committed:
MCR-2153 do not delete files from index when reindexing derivate document

MyCoRe-Travis commented:
Something gone wrong on 989cba277a679b54334812e61a06d2c64c19b004! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5268-pull_request-905

chiliego committed:
MCR-2131 URN may removed from derivate xml (#915)

When replacing a file with an urn in a derivate the urn is removed from the derivate xml

chiliego committed:
MCR-2150 URN cannot be registered (#911)
